src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/Report.kt

```diff
@@ -39,7 +39,7 @@ import kotlin.time.Duration
 
 /**
  * The message reporting feature in the bot.
- * @author NoComment1105
```

Comment:
Why no author? This extends to a few other places throughout this class.

Reply:
No author on class imo, author on functions and commabds